dotcom-rendering/src/components/InlineProductCard.stories.tsx

```diff
@@ -0,0 +1,123 @@
+import { css } from '@emotion/react';
+import { breakpoints } from '@guardian/source/foundations';
+import type { Meta } from '@storybook/react-webpack5';
+import { ArticleDesign, ArticleDisplay, Pillar } from '../lib/articleFormat';
+import type { ProductImage } from '../types/content';
+import { ArticleContainer } from './ArticleContainer';
+import type { InlineProductCardProps } from './InlineProductCard';
+import { InlineProductCard } from './InlineProductCard';
+import { Section as SectionComponent } from './Section';
+
+const meta = {
+	component: InlineProductCard,
+	title: 'Components/InlineProductCard',
+	parameters: {
+		chromatic: {
+			viewports: [
+				breakpoints.mobile,
+				breakpoints.tablet,
+				breakpoints.wide,
+			],
+		},
+
+		formats: [
+			{
+				design: ArticleDesign.Standard,
+				display: ArticleDisplay.Standard,
+				theme: Pillar.Lifestyle,
+			},
+		],
+	},
+	decorators: [
+		(Story) => (
+			<SectionComponent
+				shouldCenter={true}
+				showSideBorders={true}
+				centralBorder={'full'}
+				css={css`
+					strong {
+						font-weight: bold;
+					}
+				`}
+				format={{
+					design: ArticleDesign.Review,
+					display: ArticleDisplay.Showcase,
+					theme: Pillar.Lifestyle,
+				}}
+			>
+				<ArticleContainer
+					format={{
+						design: ArticleDesign.Review,
+						display: ArticleDisplay.Showcase,
+						theme: Pillar.Lifestyle,
+					}}
+				>
+					<Story />
+				</ArticleContainer>
+			</SectionComponent>
+		),
+	],
```

Comment:
We have a `centreColumnDecorator` which may be a bit simpler here?

```suggestion
	decorators: [centreColumnDecorator],
```

I notice that with the `Section` approach you've got a bit more padding than you would get with `centreColumnDecorator`, but that looks like more padding than you'd get in an actual article page? Happy to discuss.